### ISSUE TITLE
What are the pretrained model?

### ISSUE THREAD

**Issue by BraginIvan** [ildoonet/tf-pose-estimation#359]
As far as I understand 
- https://github.com/ildoonet/tf-pose-estimation/tree/master/models/pretrained are models trained for pose estimations from scratch (include all weights of feature extractor and stages)
- https://github.com/ildoonet/tf-pose-estimation/tree/master/models/numpy 
1. CMU is the model and weights trained in this project https://github.com/CMU-Perceptual-Computing-Lab/openpose 
2. VGG model of imagenet with weights of feature extractor. Other weights (weights of stages) are randomly initialized.
Is it correct?

**Comment by BraginIvan** [comment]
If this is true, can't  understand why if I use pre-trained weights it starts with
`epoch=0.01 step=100, 19.4667 examples/sec lr=0.000100, loss=56515.8, loss_ll=12326.2, loss_ll_paf=17200.2, loss_ll_heat=7452.26, q=39`
If not it starts from the same loss
`epoch=0.01 step=100, 19.5964 examples/sec lr=0.000100, loss=48683.4, loss_ll=11363, loss_ll_paf=15200.6, loss_ll_heat=7525.43, q=34
`
